release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -45,6 +45,10 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-8-rhcos"]
 === {op-system-first}
 
+[id="ocp-4-8-rhcos-rhel-8-4-packages"]
+==== {op-system} now supports RHEL 8.4
+
+{op-system} is now using Red Hat Enterprise Linux (RHEL) 8.4 packages. This enables you to have the latest fixes, features, and enhancements, as well as the latest hardware support and driver updates.
```

Comment:
Looking at the RN for 4.7, we pointed out that 4.6 and older releases would continue to use RHEL 8.2 packages.

I think it might be reasonable to restate that here.

Also, 4.7 will eventually upgrade to using RHEL 8.4...not sure if we want to capture that here.